.eslintrc.js

```diff
@@ -27,6 +27,14 @@ module.exports = {
         "@typescript-eslint/explicit-module-boundary-types": "off",
       },
     },
+    {
+      files: "packages/eslint-plugin-orbit-components/**/*",
```

Comment:
Best to be specific, to ensure that ESLint doesn't fail for JS files where `@typescript-eslint` plugin isn't enabled and those rules don't exist.
```suggestion
      files: "packages/eslint-plugin-orbit-components/**/*.ts?(x)",
```